src/ng/directive/ngStyle.js

```diff
@@ -54,7 +54,14 @@
 var ngStyleDirective = ngDirective(function(scope, element, attr) {
   scope.$watchCollection(attr.ngStyle, function ngStyleWatchAction(newStyles, oldStyles) {
     if (oldStyles && (newStyles !== oldStyles)) {
-      forEach(oldStyles, function(val, style) { element.css(style, '');});
+      if (!newStyles) {
+        newStyles = {};
```

Comment:
This should be tested as well. Do we have such a test already?

Reply:
You're right. I added a test case for the new model is null (c44b2f25011789183d80f13f00e8d1957c19e195).